database/reverse.go

```diff
@@ -200,6 +202,7 @@ func parseReverseSwap(rows *sql.Rows) (*ReverseSwap, error) {
 	}
 
 	reverseSwap.CreatedAt = parseTime(createdAt.Int64)
+	reverseSwap.PaidAt = parseTime(paidAt.Int64)
```

Comment:
Should we really coalesce this with 0?

Reply:
Not that critical since its just internally